resources/baseJenkins.yaml

```diff
@@ -175,8 +175,16 @@ tool:
       properties:
       - installSource:
           installers:
-          - zip:
-              url: "https://ci.opensearch.org/ci/dbc/tools/OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.zip"
+          - command:
+              command: "JENKINS_HOME_LINUX=\"/var/jenkins\"\nJENKINS_HOME_WINDOWS=\"\
+                C:/Users/Administrator/jenkins\"\n \n\nif uname -s | grep -i NT; then\n\
+                \    echo windows agent\n    JENKINS_JDK14_PATH=$JENKINS_HOME_WINDOWS/tools/hudson.model.JDK/openjdk-14\n\
+                \    JDK14_ZIP=\"OpenJDK14U-jdk_x64_windows_hotspot_14.0.2_12.zip\"\
+                \n    \nelse\n     echo linux agent\n     JENKINS_JDK14_PATH=$JENKINS_HOME_LINUX/tools/hudson.model.JDK/openjdk-14\n\
+                \     JDK14_ZIP=\"OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.zip\"\
+                \n     \nfi\n\n\nmkdir -p $JENKINS_JDK14_PATH\ncd $JENKINS_JDK14_PATH\n\
+                pwd\nrm -rf ./*\ncurl -sSL https://ci.opensearch.org/ci/dbc/tools/$JDK14_ZIP\
+                \ -o jdk14.zip\nunzip -q jdk14.zip\nrm jdk14.zip\n\n"
```

Comment:
How will this be differentiated for windows and linux since baseJenkins is same for both?

Reply:
My bad forgot that those are agents and this runs on main node.

Reply:
We use uname to differentiate between linux and windows.